Warnings in building delagent on GCC8

<!-- Before filling this issue, please read the wiki (https://github.com/fossology/fossology/wiki)
and search if the bug do not already exists in the issues (https://github.com/fossology/fossology/issues). -->

### Logs

```lisp
util.c: In function ‘deleteUpload.part.1’:
util.c:399:39: warning: ‘%s’ directive output may be truncated writing up to 1023 bytes into a region of size 1013 [-Wformat-truncation=]
       snprintf(SQL,MAXSQL,"DROP TABLE %s;", uploadtree_tablename);
                                       ^~    ~~~~~~~~~~~~~~~~~~~~
util.c:399:7: note: ‘snprintf’ output between 13 and 1036 bytes into a destination of size 1024
       snprintf(SQL,MAXSQL,"DROP TABLE %s;", uploadtree_tablename);
       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/src/delagent/agent/util.c
+++ b/src/delagent/agent/util.c
@@ -386,7 +386,7 @@ int deleteUpload (long uploadId, int userId, int userPerm)
   PQexecCheckClear("Deleting tag_uploadtree", SQL, __FILE__, __LINE__);
 
   /* Delete uploadtree_nnn table */
-  char uploadtree_tablename[1024];
+  char uploadtree_tablename[1000];
   snprintf(SQL,MAXSQL,"SELECT uploadtree_tablename FROM upload WHERE upload_pk = %ld;",uploadId);
   result = PQexec(pgConn, SQL);
   if (fo_checkPQresult(pgConn, result, SQL, __FILE__, __LINE__)) {

